misc/scripts/file_format.sh

```diff
@@ -30,13 +30,13 @@ while IFS= read -rd '' f; do
     # Ensures that files have LF line endings.
     dos2unix "$f" 2> /dev/null
     # Ensures that files do not contain a BOM.
-    sed -i '1s/^\xEF\xBB\xBF//' "$f"
+    sed -i -e '1s/^\xEF\xBB\xBF//' "$f"
```

Comment:
>-i extension
>             Edit files in-place, saving backups with the specified extension.  If a zero-length extension is given, no backup will be saved.  It is not recommended to give a zero-length extension when in-place editing files, as you risk corruption or partial content in situations where disk space is exhausted, etc.

This is what causing `-e` files, adding "" after `-i` seems to fix it.

```suggestion
    sed -i "" -e '1s/^\xEF\xBB\xBF//' "$f"
```
but it won't work with GNU sed.